### PR TITLE
Release post for v4.2.1

### PR DESCRIPTION
**Pull request by ashmaroli**
- This is a 🔦 documentation change.

## Summary

Time to release `v4.2.1` from `4.2-stable` branch.

### CODE REVIEW

File: docs/_posts/2021-09-20-jekyll-4-2-1-released.markdown
Comment (mattr-) [inline docs/_posts/2021-09-20-jekyll-4-2-1-released.markdown:24]:
```suggestion
{% raw %}`{{ page.content }}`{% endraw %} and other mutable attributes from reflecting the
```

File: docs/_posts/2021-09-20-jekyll-4-2-1-released.markdown
Comment (mattr-) [inline docs/_posts/2021-09-20-jekyll-4-2-1-released.markdown:18]:
```suggestion
unforeseen consequence &mdash; absolute URLs for assets now pointed to
```

having `unforeseen` and `undesired` together feels weird when reading this, so i took out `undesired` since the fact that we changed it made it obvious it was a undesired change. 😄

**Review by mattr- — COMMENTED** [review]
I was going to suggest a 4.2.1 release soon. Thanks for writing this post and prepping the release. ❤️ 

I left a couple of language nits to consider.

**Review by mattr- — APPROVED** [review]
:shipit:

**Comment by ashmaroli** [comment]
@mattr- Is anything more needed to ship `v4.2.1`?

**Comment by mattr-** [comment]
I'm not aware of anything, so let's cut the release and 🚢  all the things

**Comment by ashmaroli** [comment]
I don't have the access on Rubygems. So its either you or @parkr..

**Comment by ashmaroli** [comment]
Note: the patches are on the `4.2-stable` branch..

**Review by parkr — APPROVED** [review]

**Comment by parkr** [comment]
You now have push access to the Jekyll gem!

**Comment by ashmaroli** [comment]
> You now have push access to the Jekyll gem!

Yep. I'm waiting for the documentation on cutting releases from a `*-stable` branch.

**Comment by ashmaroli** [comment]
@jekyllbot: merge +doc

**Pull request merged**
